tests/e2e/test_main.py

```diff
@@ -198,23 +198,26 @@ def test_image_build_custom_preset(
     img_name = f"image:extras-e2e-custom-preset-{rnd}"
     img_uri_str = f"{img_name}:{tag}"
 
-    result = cli_runner(
-        [
-            "neuro",
-            "image-build",
-            "--preset",
-            custom_preset,
-            "-f",
-            str(dockerfile_path),
-            ".",
-            img_uri_str,
-        ]
-    )
-    assert result.returncode == 0, result
-    sleep(10)
+    try:
+        result = cli_runner(
+            [
+                "neuro",
+                "image-build",
+                "--preset",
+                custom_preset,
+                "-f",
+                str(dockerfile_path),
+                ".",
+                img_uri_str,
+            ]
+        )
+        assert result.returncode == 0, result
+        sleep(10)
```

Comment:
by the way: together with `repeat_until_success`, we don't need `sleep(10)` anymore